### PR TITLE
Add Somerville, MA

### LINKED ISSUE #618: Somerville, MA
To: cityclerk@somervillema.gov, citycouncil@somervillema.gov, mayor@somervillema.gov
Subject: Demanding support for defunding the police in Somerville

Message: 

Dear Somerville City Councilors,

My name is [YOUR NAME], and I am a resident of Somerville. This past week, our nation has been gripped by protests calling for a rapid and meaningful reconsideration of the role of policing in communities as well as an end to racism and anti-Blackness in America. Our city has been deeply engaged in much of this action. Accordingly, it has come to my attention that the city plans to adopt a number of the reform proposals created by Campaign Zero. I ask that you reconsider these attempts to reform and instead pivot your attention to defunding the police for fiscal year 2021.

SPD has been a waste of our resources. In fiscal year 2020, the SPD budget was approximately 18% of the total—less only than schools. The city of Somerville will spend over $17,044,021 on the Police Department this year, as you well know. By contrast, Health and Human Services receives only $3,064,691. This is an abomination. Budgets for Immigrant services, disability services, and fair housing services also pale in comparison to this excessive budget While we’ve been spending extraordinary amounts on policing, we have not seen improvements to safety, homelessness, mental health, or affordability in our city. Instead, we see wasteful and harmful actions of our police.

I call on you to defund the SPD and instead meaningfully reallocate funds towards social programs and resources that support housing, jobs, education, health care, child care, and other critical community needs. I demand a budget that supports community wellbeing, rather than empowers the police forces that tear them apart.

We can be a beacon for other cities to follow if only we have the courage to change.

Sincerely,
[YOUR NAME] [YOUR ADDRESS] [YOUR EMAIL] [YOUR PHONE NUMBER]

### PR DESCRIPTION
**Pull request by DMcNamara**
Closes #618

### CODE REVIEW

File: _emails/somerville.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+permalink: "/somerville"
+name: Letter to Council Members
+city: Somerville
+state: MA
+recipients:
+- cityclerk@somervillema.gov
+- citycouncil@somervillema.gov
+subject: Demanding support for defunding the police in Somerville
+body: |-
+Dear Somerville City Councilors,
```
Comment (minicreative) [inline _emails/somerville.md:12]: Please put 2 spaces before each line in the body (this will break if merged). Thank you

Comment (DMcNamara) [reply _emails/somerville.md]: Should be fixed now, sorry I missed that.

**Comment by NickCrews** [comment]
Looks good to me now.

File: _emails/somerville.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  I call on you to defund the SPD and instead meaningfully reallocate funds towards social programs and resources that support housing, jobs, education, health care, child care, and other critical community needs. I demand a budget that supports community wellbeing, rather than empowers the police forces that tear them apart.
+
+  We can be a beacon for other cities to follow if only we have the courage to change.
+
+  Sincerely,
+
+  \[YOUR NAME\]
+  \[YOUR ADDRESS\]
+  \[YOUR EMAIL\]
+  \[YOUR PHONE NUMBER\]
```
Comment (Ceshion) [inline _emails/somerville.md:30]: Please remove bracket escaping here, as this is not necessary for them to render correctly

File: _emails/somerville.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+state: MA
+recipients:
+- cityclerk@somervillema.gov
+- citycouncil@somervillema.gov
+- mayor@somervillema.gov
+subject: Demanding support for defunding the police in Somerville
+body: |-
+  Dear Mayor Curtatone and Somerville City Councilors,
+
+  My name is \[YOUR NAME\], and I am a resident of \[CITY\]. This past week, our nation has been gripped by protests calling for a rapid and meaningful reconsideration of the role of policing in communities as well as an end to racism and anti-Blackness in America. Our city has been deeply engaged in much of this action. Accordingly, it has come to my attention that the city plans to adopt a number of the reform proposals created by Campaign Zero. I ask that you reconsider these attempts to reform and instead pivot your attention to defunding the police for fiscal year 2021.
```
Comment (Ceshion) [inline _emails/somerville.md:15]: Please also remove bracket escaping here

**Comment by DMcNamara** [comment]
@Ceshion Removed. copied those from the oakland email since it was listed as a good example.

**Pull request merged**